_bibliography/references.bib

```diff
@@ -1,3 +1,15 @@
+@inproceedings{ Jain-2023-ml_llvm_tools,
```

Comment:
Please change it to Venkatakeerthy-2025-VexIR2Vec

Reply:
Also please use the bibtex entry in the ACM publication page. You may remove the abstract and note from that entry. Use the tag I mentioned above.